src/selectors/lines/transforms.js

```diff
@@ -3,31 +3,39 @@
 import type { ResmokeLog } from 'src/models';
 
 export function parseLogLine(line: string): string {
-    let logParts = line.split('|'); // in many cases mongod will insert a pipe between the metadata and json logs
-    if (logParts.length !== 2) {
-      const startOfJson = line.indexOf('{'); // if not, attempt to find the first occurence of a json document and attempt to parse as a log
-      if (startOfJson > -1) {
-        logParts = [line.substring(0, startOfJson), line.substring(startOfJson)];
-      } else {
-        return line;
-      }
-    }
-    const structedLog = parseMongoJson(logParts[1]);
-    if (structedLog === null) {
+  let logParts = line.split('|'); // in many cases mongod will insert a pipe between the metadata and json logs
+  if (logParts.length !== 2) {
+    const startOfJson = line.indexOf('{'); // if not, attempt to find the first occurence of a json document and attempt to parse as a log
+    if (startOfJson > -1) {
+      logParts = [line.substring(0, startOfJson), line.substring(startOfJson)];
+    } else {
       return line;
     }
-    return `${logParts[0]}| ${ structedLog.t.$date } ${ structedLog.s.padEnd(2) } ${ structedLog.c.padEnd(8)} ${ structedLog.id.toString().padEnd(7)} [${ structedLog.ctx }] ${ JSON.stringify(structedLog.msg) }${ structedLog.attr ? ',"attr":' + JSON.stringify(structedLog.attr) : '' }`;
+  }
+  const structedLog = parseMongoJson(logParts[1]);
+  if (structedLog === null) {
+    return line;
+  }
+  return `${logParts[0]}| ${structedLog.t.$date} ${structedLog.s.padEnd(
+    2
+  )} ${structedLog.c.padEnd(8)} ${structedLog.id.toString().padEnd(7)} [${
+    structedLog.ctx
+  }] ${JSON.stringify(structedLog.msg)}${
+    structedLog.t ? ',"t":' + JSON.stringify(structedLog.t) : ''
```

Comment:
@john-m-liu  the changes are here they blended in because of the formatting changes. I added the `structedLog.t` field and the `strucuredLog.s` field

Reply:
Those fields were already there. The ticket is asking for the "truncated" and "size" fields

Reply:
🤦 thanks idk how i didnt realize that. When i looked at the file i  thought `.t` and `.s` were just abbreviations for truncated and size